client/blocks/comments/post-comment-content.jsx

```diff
@@ -21,21 +38,25 @@ export default class PostCommentContent extends React.Component {
 				</div>
 			);
 		}
-
 		/*eslint-disable react/no-danger*/
 		return (
 			<AutoDirection>
-				<div
-					className="comments__comment-content"
-					dangerouslySetInnerHTML={ { __html: this.props.content } }
-				/>
+				<div className={ classNames( 'comments__comment-content-wrapper', this.props.className ) }>
+					<div
+						className="comments__comment-content"
+						ref="overflowNode"
```

Comment:
Can't. At least not afaict. The OverflowContainer needs to be able to find the overflow ref in the child. I tried setting a prop on the child via `this.overflowNode = node` in a ref handler, but then the OverflowContainer can't see the nested ref...